src/styles/blocks/review.scss

```diff
@@ -0,0 +1,16 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
review is a div. it will be 100% wide automatically

Reply:
If I don't set the width to 100% it won't be positioned properly